test/configCases/target/node-dynamic-import/test.filter.js

```diff
@@ -0,0 +1,5 @@
+var supportsES6 = require("../../../helpers/supportsES6");
```

Comment:
That's the wrong test. node 0.12 supports ES6 classes, but no arrow function.

So you need to add a new helper (i. e. supportsArrowFunction) to test correctly.